### DISCUSSION TITLE
Clipping frame visbility

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#4020] [General]
I would like to find out how I can turn off the clipping frame and then turn it back on.  I have implemented the Section Menu in my app 

![ClipMenu](https://user-images.githubusercontent.com/83007711/182796922-3df4c195-bb58-466c-ad5d-0ed2a1c0038a.JPG)

For the image below, I used the Shape option

![ClipBox](https://user-images.githubusercontent.com/83007711/182796991-19bcd9db-80c6-48b0-a544-61766fac0ff6.JPG)

I want to turn off the dynamic clipping frame so that user only sees clipped area (without the frame) - as  shown on right and then turn the frame back - as shown on left - on so that user can change the clipped region.

Can you please guide how I can achieve this functionality

Thank you,

Ovais

**Comment by pmconne** [comment]
```
const vp = IModelApp.viewManager.selectedView;
if (vp) {
  const provider = ViewClipDecorationProvider.create();
  provider.showDecoration(vp);
}
```
